app/src/main/res/values-vi/strings.xml

```diff
@@ -211,16 +211,16 @@
     <string name="the_number_of_days_allowed">Số ngày xin nghỉ không được để trống!</string>
     <string name="the_field_required_can_not_be_blank">Trường bắt buộc không được để trống!</string>
     <string name="time_must_be_a_working_date">Thời gian phải là một ngày làm việc!</string>
-    <string name="annual_leave_is_not_greater_than_14_5_day">Nghỉ phép thường niên không được quá 14,5 ngày!</string>
-    <string name="leave_for_marriage_is_not_greater_than_3_day">Nghỉ kết hôn không được quá 3 ngày!</string>
-    <string name="leave_for_child_marriage_is_not_greater_than_1_day">Nghỉ con kết hôn không được quá 1 ngày!</string>
-    <string name="funeral_leave_is_not_greater_than_3_day">Nghỉ lễ tang không được quá 3 ngày!</string>
-    <string name="leave_for_care_of_sick_is_not_greater_than_20_day">Nghỉ con ốm không được quá 20 ngày!</string>
-    <string name="pregnacy_examination_leave_is_not_greater_than_14_5_day">Nghỉ khám thai không được quá 5 ngày!</string>
-    <string name="sick_leave_is_not_greater_than_60_day">Nghỉ đau ốm không dduwuocj quá 60 ngày!</string>
-    <string name="miscarriage_leave_is_not_greater_than_50_day">Nghỉ sảy thai không được quá 50 ngày!</string>
-    <string name="maternity_leave_is_not_greater_than_180_day">Nghỉ thai sản không được quá 180 ngày!</string>
-    <string name="wife_labor_leave_is_not_greater_than_14_day">Nghỉ vợ sinh không được quá 14 ngày!</string>
+    <string name="annual_leave_error">Nghỉ phép thường niên không được quá 14.5 ngày!</string>
```

Comment:
ERROR: [StringFormatInvalid] Format string '`annual_leave_error`' is not a valid format string so it should not be passed to `String.format`
 If a string contains a '%' character, then the string may be a formatting string which will be passed to `String.format` from Java code to replace each '%' occurrence with specific values.

This lint warning checks for two related problems:
(1) Formatting strings that are invalid, meaning that `String.format` will throw exceptions at runtime when attempting to use the format string.
(2) Strings containing '%' that are not formatting strings getting passed to a `String.format` call. In this case the '%' will need to be escaped as '%%'.

NOTE: Not all Strings which look like formatting strings are intended for use by `String.format`; for example, they may contain date formats intended for `android.text.format.Time#format()`. Lint cannot always figure out that a String is a date format, so you may get false warnings in those scenarios. See the suppress help topic for information on how to suppress errors in that case.